src/app/containers/Ad/utilities/adSlotStyles.js

```diff
@@ -80,12 +87,17 @@ export const leaderboardStyles = css`
 export const mpuStyles = css`
   display: none;
   visibility: hidden;
+  padding: 0;
   @media (min-width: ${MPU_WIDTH_MIN}) {
     min-height: ${MPU_HEIGHTS.GROUP_1};
-    padding: 0 ${GEL_SPACING};
-    display: block;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
     visibility: visible;
   }
+  @media (min-width: ${MPU_WIDTH_MIN_PADDING}) {
+    padding: 0 ${GEL_SPACING};
```

Comment:
This padding seems to overriden by another one:
<img width="484" alt="Screenshot 2020-08-19 at 15 08 04" src="https://user-images.githubusercontent.com/46446236/90645521-f03a9c00-e22d-11ea-8ed3-eef5d5c55acb.png">

Reply:
Not sure if we need it though, since we have a left and right margin of 8px applied to the page container.